app/api/v2/versions.py

```diff
@@ -71,6 +72,18 @@ def get_version(
     atbd = crud_atbds.get(db=db, atbd_id=atbd_id, version=major)
     [atbd_version] = atbd.versions
     check_permissions(principals=principals, action="view", acl=atbd_version.__acl__())
+    # Hide reviewer info from atbd_version for other users
+    if not check_permissions(
+        principals=principals,
+        action="view_update_reviewer_info",
+        acl=atbd_version.__acl__(),
+        raise_exception=False,
+    ):
+        atbd_version.contacts_link = [
+            contact
+            for contact in atbd_version.contacts_link or []
+            if RolesEnum.DOCUMENT_REVIEWER.value not in contact.roles or []
+        ]
```

Comment:
We also have to handle the case where a contact has multiple roles but one of them is `DOCUMENT_REVIEWER`. In that case, I think we need to hide the `DOCUMENT_REVIEWER` role instead of hiding the contact all together.

Reply:
Can you check again? The above logic does that.

Current schema
```
contacts_link: ContactsLinkOutput[]

ContactsLinkOutput:
    contact: ContactsSummary
    roles: Optional[List[RolesEnum]] = []
    ...
```

I have updated the variable naming.

Reply:
New commit https://github.com/NASA-IMPACT/nasa-apt/pull/817/commits/48f9642fdcbb10b85fa73a4c3c00bbf7dff73ae2